### PR TITLE
add support to react to callback query with no notification

### LINKED ISSUE #192: Support no notification option for answerCallbackQuery
According to [CallbackQuery docs](https://core.telegram.org/bots/api#callbackquery)

> NOTE: After the user presses a callback button, Telegram clients will display a progress bar until you call answerCallbackQuery. It is, therefore, necessary to react by calling answerCallbackQuery even if no notification to the user is needed (e.g., without specifying any of the optional parameters).

### PR DESCRIPTION
**Pull request by eanea**
Closes #192

### CODE REVIEW

File: core/shared/src/main/scala/canoe/api/models/CallbackQueryAPI.scala [hunk 1]
```diff
@@ -23,6 +23,12 @@ final class CallbackQueryAPI(private val query: CallbackQuery) extends AnyVal {
   def alert[F[_]: TelegramClient](text: String): F[Boolean] =
     AnswerCallbackQuery.alert(query.id, text).call
 
+  /**
+    * React with no notification
+    */
+  def withoutNotification[F[_]: TelegramClient]: F[Boolean] =
```
Comment (augustjune) [inline core/shared/src/main/scala/canoe/api/models/CallbackQueryAPI.scala:29]: Let's rename `withoutNotification` to `finish`. It follows the pattern of using verbs to denote actions like `query.alert`, `query.inform`, `query.redirect`. Subjectively, it gives a better intuition of the underlying action.

Please compare:
```
for {
  _ <- query.message.traverse(_.chat.send(cbd))
  _ <- query.finish
}
```
vs.
```
for {
  _ <- query.message.traverse(_.chat.send(cbd))
  _ <- query.withoutNotification
}
```

**Review by augustjune — CHANGES_REQUESTED** [review]

**Comment by augustjune** [comment]
Hi @eanea!
Thanks for your contribution, I think it will be useful for a lot of people using callback query API 👍

File: core/shared/src/main/scala/canoe/api/models/CallbackQueryAPI.scala (hunk 1 above)
Comment (eanea) [reply core/shared/src/main/scala/canoe/api/models/CallbackQueryAPI.scala]: @augustjune, agree. looks good.

**Comment by eanea** [comment]
@augustjune, done.

**Review by augustjune — APPROVED** [review]

**Comment by augustjune** [comment]
Nice 👍

**Pull request merged**